### ISSUE TITLE
python-is-python2 is deprecated

### ISSUE THREAD

**Issue by HeRo002** [kees-z/DuplicatiDocs#65]
This issue is a consequence of https://github.com/duplicati/duplicati/issues/4316 - "python-is-python2 is deprecated and can break modern software #4316". 

When that is solved, the manual should be changed so that the package python-is-python2 is NOT installed under Ubuntu (and possibly other linux versions). See the manual here: https://duplicati.readthedocs.io/en/latest/02-installation/ . 

The 'python-is-python2' package is not directly mentioned in the Ubuntu commands, but it gets installed anyway. The following command in Terminal: 

:~$ sudo apt install apt-transport-https sudo nano git-core python software-properties-common dirmngr

Gives this output (among a lot of other output of course):

Note, selecting 'python-is-python2' instead of 'python'

There are other things that need updating in the manual, as mentioned in https://forum.duplicati.com/t/installation-instructions-need-update/10459 .

**Comment by HeRo002** [comment]
Someone closed the other issue duplicati/duplicati#4316 - "python-is-python2 is deprecated and can break modern software #4316". But as far as I can tell, it is not solved - see my comment there.

**Comment by kees-z** [comment]
Feel free to submit a PR, so this can be properly added to the docs.

**Comment by jfparis** [comment]
Hello

A couple of months ago, I ported the python utilities to python3. I also followed best practices so all in all the run both with python2 and python3.
Canary versions from 2.0.5.113 do not depend on python2 to run the `DuplicatiVerify.py` tool 

The question is when to update the doc (and simply remove "python" from the list of dependencies) in the following line 
`sudo apt install apt-transport-https nano git-core python software-properties-common dirmngr -y`

This could be done
* Either wait for 113 or higher to be promoted as beta
* Do it right away. In that case, any user wanting to run the DuplicatiVerify.py would run into a problem but let's be honest this is a hedge case.

I am not the doc maintainer so this is not my decision but I thought the above would be useful in the discussion

Also useful, I have verified that one can install and run duplicati without the `python` package installed
